LibTest/io/Stdin/first_A04_t01.dart

```diff
@@ -43,7 +43,10 @@ run_main() async {
     await process.stdout.toList().then((out) { res_elementAt = out[0]; });
```

Comment:
Same problem as above, `out` is a `List<List<int>>` there are no guarantees that `out[0]` will contain all the bytes.